datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -5674,6 +5674,11 @@ type DataJob implements EntityWithRelationships & Entity & BrowsableEntity {
     Whether or not this entity exists on DataHub
     """
     exists: Boolean
+
+    """
+    fine grained lineage
```

Comment:
Minor nit: comment could be improved. What is fine grained lineage?

Reply:
lol very good call out that's a weak comment up there